iac-common/src/main/java/org/sonar/iac/common/yaml/object/AttributeObject.java

```diff
@@ -50,6 +50,10 @@ public AttributeObject reportIfValue(Predicate<YamlTree> predicate, String messa
     return this;
   }
 
+  public boolean isAbsentOrEmpty(Predicate<YamlTree> predicate) {
+    return this.status == Status.ABSENT || (tree != null && predicate.test(tree.value()));
+  }
```

Comment:
I thought about this method, and while it's working really well how you implemented it, it deviates from the way we raise issues in the other checks. I will write some suggestion in slack how this can be changed.